src/main/kotlin/app/revanced/patches/youtube/misc/playercontrols/PlayerControlsBytecodePatch.kt

```diff
@@ -0,0 +1,69 @@
+package app.revanced.patches.youtube.misc.playercontrols
+
+import app.revanced.extensions.exception
+import app.revanced.patcher.data.BytecodeContext
+import app.revanced.patcher.extensions.InstructionExtensions.addInstruction
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint.Companion.resolve
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprintResult
+import app.revanced.patcher.patch.BytecodePatch
+import app.revanced.patcher.patch.annotation.Patch
+import app.revanced.patches.shared.fingerprints.LayoutConstructorFingerprint
+import app.revanced.patches.youtube.misc.playercontrols.fingerprints.BottomControlsInflateFingerprint
+import app.revanced.patches.youtube.misc.playercontrols.fingerprints.PlayerControlsVisibilityFingerprint
+import com.android.tools.smali.dexlib2.iface.instruction.OneRegisterInstruction
+
+@Patch(
+    description = "Manages the code for the player controls of the YouTube player.",
+    dependencies = [BottomControlsResourcePatch::class],
+)
```

Comment:
❓ : Missing patch name

```suggestion
@Patch(
    name = "Player controls bytecode patch",
    description = "Manages the code for the player controls of the YouTube player.",
    dependencies = [BottomControlsResourcePatch::class],
)
```